src/doc/reference.md

```diff
@@ -2070,6 +2070,9 @@ macro scope.
    trait of the same name. `{Self}` will be replaced with the type that is supposed
    to implement the trait but doesn't. To use this, the `on_unimplemented` feature gate
    must be enabled.
+- `must_use` - on structs and enums, will warn if this value isn't bound to a variable.
```

Comment:
This has little to do with binding to variables - after all, `.unwrap()` makes the warning go away